src/AspNet.Security.OAuth.DingTalk/DingTalkAuthenticationHandler.cs

```diff
@@ -69,9 +69,9 @@ protected override async Task<OAuthTokenResponse> ExchangeCodeAsync([NotNull] OA
             });
 
             using var request = new HttpRequestMessage(HttpMethod.Post, address);
-            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue("application/json"));
-            using var requestContent = await CopyPayloadAsync(new Dictionary<string, StringValues> { { "tmp_auth_code", tokens.AccessToken } });
-            request.Content = new StringContent(requestContent.RootElement.ToString(), System.Text.Encoding.UTF8, "application/json");
+            request.Headers.Accept.Add(new MediaTypeWithQualityHeaderValue(System.Net.Mime.MediaTypeNames.Application.Json));
```

Comment:
Add the `System.Net.Mime` namespace into the usings, then these usages can just be `MediaTypeNames.Application.Json`.

Reply:
ok